modules/con_network-observability-ipsec.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-ipsec_{context}"]
+= Working with IPsec
+In {product-title}, IPsec is disabled by default. You can enable IPsec by following the instructions in "Configuring IPsec encryption."
+
+When IPsec is enable, a new column named *IPSec Status* is displayed in the network observability Traffic flows view to show whether a flow was successfully IPSec-encrypted or if there was an error during encyption/decryption.
```

Comment:
Would be worth to mention the new metric displayed in the netobserv health dashboard here.

PR is not yet merged: https://github.com/netobserv/network-observability-operator/pull/1573